### PR TITLE
Docs/r3.3 3548 hydrator docs

### PR DESCRIPTION
**Pull request by awholegunch**
This PR removes the original Hydrator documentation from CDAP, and switches to using the caskdata/hydrator-plugins repo.

[Quick Build](http://builds.cask.co/browse/CDAP-DOB85-5)

Build fails with three known errors. These are OK to accept and will be fixed in later PRs.

However, there is a change to review in this PR. In the original docs, we had a table showing available plugins; I have replaced this table with a link to a list (table of contents). The reason for doing so is that this list is automatically generated by Sphinx. Once a document is added to the build.sh to be downloaded to a directory, it will then be automatically included in the list and the documentation with no further entries. This makes updating simple.

The table was in the section titled "System Artifacts, ETL Applications, and ETL Plugins".

The original page with table: [A Quick Intro to ETL](http://docs.cask.co/cdap/3.3.0-SNAPSHOT/en/cdap-apps/hydrator/overview.html#a-quick-intro-to-etl)
The revised page with just a link:  [A Quick Intro to ETL](http://builds.cask.co/artifact/CDAP-DOB85/shared/build-5/Docs-HTML/3.3.0-SNAPSHOT/en/cdap-apps/hydrator/overview.html#a-quick-intro-to-etl)

New list of plugins: [ETL Plugins](http://builds.cask.co/artifact/CDAP-DOB85/shared/build-5/Docs-HTML/3.3.0-SNAPSHOT/en/cdap-apps/hydrator/hydrator-plugins/index.html)

Fix for https://issues.cask.co/browse/CDAP-3548

### CODE REVIEW

File: cdap-docs/admin-manual/text.txt [hunk 1]
```diff
@@ -0,0 +1,2 @@
+# JavaScript
+# JavaScriptto be prepended
```
Comment (albertshau) [inline cdap-docs/admin-manual/text.txt]: is this file and cdap-docs/admin-manual/test.txt supposed to be here?

File: cdap-docs/cdap-apps/source/_includes/hydrator-plugins/third-party.rst [hunk 2]
```diff
@@ -26,16 +26,16 @@ A sample JDBC Driver Plugin configuration:
       "parents": [ "cdap-etl-batch[|version|,\ |version|]" ],
       "config": [
         {
-          "name" : "mysql",
-          "type" : "jdbc",
-          "className" : "com.mysql.jdbc.Driver",
-          "description" : "Plugin for MySQL JDBC driver"
+          "type" : "JDBC",
```
Comment (albertshau) [inline cdap-docs/cdap-apps/source/_includes/hydrator-plugins/third-party.rst]: these cannot be changed. The type and name are very specific.

File: cdap-docs/cdap-apps/source/_includes/hydrator-plugins/third-party.rst [hunk 3]
```diff
@@ -26,16 +26,16 @@ A sample JDBC Driver Plugin configuration:
...
+          "description" : "Plugin for MySQL JDBC driver",
+          "className" : "com.mysql.jdbc.Driver"
         },
         {
-          "name" : "postgresql",
-          "type" : "jdbc",
-          "className" : "org.postgresql.Driver",
-          "description" : "Plugin for PostgreSQL JDBC driver"
+          "type" : "JDBC",
+          "name" : "PostgreSQL JDBC",
```
Comment (albertshau) [inline cdap-docs/cdap-apps/source/_includes/hydrator-plugins/third-party.rst]: same here

**Comment by albertshau** [comment]
a question and couple comments, otherwise lgtm

File: cdap-docs/admin-manual/text.txt (hunk 1 above)
Comment (awholegunch) [inline cdap-docs/admin-manual/text.txt]: Nope! Accidentally included. Thanks for catching them.

**Comment by albertshau** [comment]
lgtm

**Comment by anwar6953** [comment]
could you squash commits before merging please.

**Comment by awholegunch** [comment]
@anwar6953 I don't think I can as I have already merged release/3.3 into this branch. :(

**Comment by anwar6953** [comment]
Right, it'll be more difficult to squash since you've merged release/3.3 into this branch several times, and there have been conflict resolutions during the merge processes.

**Pull request merged**